### PR TITLE
fix(adminmanual): Recommend basic auth over NTLM for Exchange

### PR DESCRIPTION
**Pull request by ChristophWurst**
### ☑️ Resolves

Follow-up to https://github.com/nextcloud/documentation/pull/10452

### 🖼️ Screenshots

Bildschirmfoto vom 2023-05-26 12-43-36

### CODE REVIEW

File: admin_manual/configuration_server/email_configuration.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ connect Nextcloud to a remote SMTP server:
 .. warning:: There were changes to the 3rd party mailer library in Nextcloud 26:
     
     * STARTTLS cannot be enforced. It will be used automatically if the mail server supports it. The encryption type should be set to 'None' in this case. See :ref:`here<TLSPeerVerification>` for an example on how to configure self signed certificates.
-    * NTLM authentication for Microsoft Exchange is not supported by the new mailer library
+    * NTLM authentication for Microsoft Exchange is not supported by the new mailer library. Use `basic authentication <https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/authentication-and-ews-in-exchange#basic-authentication>`_.
```
Comment (nickvergessen) [inline admin_manual/configuration_server/email_configuration.rst:41]:
```suggestion
    * NTLM authentication for Microsoft Exchange is not supported by the new mailer library. Try `basic authentication <https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/authentication-and-ews-in-exchange#basic-authentication>`_ instead.
```

**Comment by kesselb** [comment]
image

- Added Joas suggestion
- Resolved the conflict

**Review by kesselb — APPROVED** [review]

**Pull request merged**